app/views/provider_interface/interviews/_form.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= f.govuk_error_summary %>
+
+<%= f.govuk_date_field :date, legend: { size: 'm' } %>
+
+<%= f.govuk_text_field :time, width: 5, label: { size: 'm' } %>
+
+<% if @wizard.multiple_application_providers? %>
+  <%= f.govuk_radio_buttons_fieldset :provider_id, legend: { text: 'Organisation carrying out interview', size: 'm' } do %>
+    <% @wizard.application_providers.each do |provider| %>
+      <%= f.govuk_radio_button :provider_id, provider.id, label: { text: provider.name } %>
+    <% end %>
+  <% end %>
+<% end %>
+
+<%= f.govuk_text_area :location, label: { size: 'm' } %>
+
+<%= f.govuk_text_area :additional_details, label: { size: 'm' } %>
+
+<%= f.govuk_submit 'Continue' %>
```

Comment:
```suggestion
<%= f.govuk_submit t('continue') %>
```

Reply:
:+1: https://github.com/DFE-Digital/apply-for-teacher-training/pull/3993/commits/555105222e11aecf8ac7e144ea254a76419cc838